tfhe/src/integer/ciphertext/utils.rs

```diff
@@ -0,0 +1,101 @@
+use super::{CrtCiphertext, RadixCiphertext, SignedRadixCiphertext};
+use crate::shortint::Ciphertext;
+
+pub trait IntegerCiphertext: Clone {
+    fn blocks(&self) -> &[Ciphertext];
```

Comment:
shouldn't this be in the mod.rs itself ?

Reply:
I think it's clearer to have the mod only containing other modules (and possibly reexports)
Renamed module `integer_ciphertext`